### PR TITLE
Basic Input structure

### PR DESCRIPTION
**Pull request by denbalogh**

### CODE REVIEW

File: storybooknative/src/InputField/InputField.stories.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// @flow
+
+/* eslint-disable jsx-a11y/accessible-emoji, import/no-unresolved, import/extensions */
```
Comment (tbergq) [inline storybooknative/src/InputField/InputField.stories.js]: Why do we need this eslint-disables?

File: storybooknative/src/InputField/index.js [hunk 2]
```diff
@@ -1,30 +1,113 @@
...
-  const { label, ...rest } = props;
+  const {
+    size = SIZE_OPTIONS.NORMAL,
+    type = TYPE_OPTIONS.TEXT,
+    label,
+    help,
+    error,
+    inlineLabel,
+    prefix,
+    sufix,
```
Comment (tbergq) [inline storybooknative/src/InputField/index.js]: suffix?

File: storybooknative/src/InputField/index.js.flow [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from "react";
+
+import Text from "../Text";
+
+export type Props = {|
+  +size?: "small" | "normal",
+  +type?: "text" | "number" | "email" | "password",
+  +defaultValue?: string,
```
Comment (wEprous) [inline storybooknative/src/InputField/index.js.flow]: What's the reason to have `defaultValue` and `value` at one time? 🙂

Comment (wEprous) [reply storybooknative/src/InputField/index.js.flow]: Thanks @tbergq :) I thought that it has some use case 👍

File: storybooknative/src/InputField/index.js [hunk 4]
```diff
@@ -1,30 +1,113 @@
 // @flow strict
 
 import * as React from "react";
-import { TextInput } from "react-native"; // eslint-disable-line no-restricted-imports
+import { TextInput, View } from "react-native"; // eslint-disable-line no-restricted-imports
```
Comment (tbergq) [inline storybooknative/src/InputField/index.js]: I don't think you have restricted imports for this project. You can delete this eslint-disable

File: storybooknative/src/InputField/index.js [hunk 5]
```diff
@@ -1,30 +1,113 @@
 // @flow strict
```
Comment (tbergq) [inline storybooknative/src/InputField/index.js]: Also, there is no flow strict setup for orbit, so `@flow` will suffice.

**Pull request merged**